plugins/woocommerce/src/Blocks/BlockTypes/OrderConfirmation/AdditionalFields.php

```diff
@@ -35,8 +35,8 @@ protected function render_content( $order, $permission = false, $attributes = []
 		$content   .= $this->render_additional_fields(
 			$controller->filter_fields_for_order_confirmation(
 				array_merge(
-					$controller->get_order_additional_fields_with_values( $order, 'contact', '', 'view' ),
-					$controller->get_order_additional_fields_with_values( $order, 'additional', '', 'view' ),
+					$controller->get_order_additional_fields_with_values( $order, 'contact', 'additional', 'view' ),
+					$controller->get_order_additional_fields_with_values( $order, 'additional', 'additional', 'view' ),
```

Comment:
I'm a little worried about how confusing this is, having two args the same, but meaning different things. Since we landed on using `Billing/Shipping/Other` in [this comment](https://github.com/woocommerce/woocommerce/discussions/42995#discussioncomment-9010273) I'm wondering if we should update the code to reflect this change.

Reply:
That would be in a follow up PR.